pylint/checkers/utils.py

```diff
@@ -352,13 +352,17 @@ def __init__(self, index):
         Exception.__init__(self, index)
         self.index = index
 
-def parse_format_string(format_string: str) -> Tuple[Set[str], int]:
+def parse_format_string(format_string: str) -> Tuple[Set[str], int,
```

Comment:
The formatting looks a bit weird, can't we put everything on the same line?

Reply:
Done.